src/app/settings/initial-settings.ts

```diff
@@ -6,6 +6,10 @@ export interface Settings extends DimApiSettings {
   readonly organizerColumnsGhost: string[];
   readonly loMinPower: number;
   readonly loMinStatTotal: number;
+  /** Whether to ignore element affinity on armor in the loadout optimizer. */
+  readonly loIgnoreAffinity: boolean;
```

Comment:
You need a corresponding PR in https://github.com/DestinyItemManager/dim-api for these.

Reply:
Ah, thought `loMinPower` & `loMinStatTotal` were in this export because they weren't exported from `dim-api`.
Would these additions then be unnecessary here?

Reply:
From memory they allow you do build stuff without needing an update to dim-api first. They could be left overs that were never cleaned up properly.

Reply:
We still need this dim-api PR for the extra setting.